lib/rtosc/.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: cpp
+
+compiler:
+    - gcc
+    - clang
+
+before_install:
+    - if [[ $NEW_GCC == true ]]; then sudo apt-get --purge remove gcc; fi;
+    - if [[ $NEW_GCC == true ]]; then sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test; fi;
+    - if [[ $NEW_GCC == true ]]; then sudo apt-get -qq update; fi;
+    - if [[ $NEW_GCC == true ]]; then sudo apt-get -qq install g++-4.8; fi;
+    - if [[ $NEW_GCC == true ]]; then sudo apt-get -qq install gcc-4.8; fi;
+    - if [[ $NEW_GCC == true ]]; then sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 90; fi;
+    - if [[ $NEW_GCC == true ]]; then sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 90; fi;
```

Comment:
Is this file used? I am assuming not in this instance and is included for completeness.